client/default.config.js

```diff
@@ -31,5 +31,11 @@ module.exports = {
             DASHBOARD_WIDGET: 'Dashboard',
             TRACKER_DASHBOARD_WIDGET: 'Tracker Dashboard',
         },
+        appChannelToDisplayName: {
+            Stable: 'Stable',
+            Development: 'Development',
+            Canary: 'Canary',
+        },
```

Comment:
I think the canonical names should be lowercase, it's weird to have a proper-cased enum.

Reply:
Agreed.